src/views/communities/Stories.tsx

```diff
@@ -100,7 +99,7 @@ export default function Stories() {
                             letterSpacing: 0.366667,
                         }}
                     >
-                        {i18n.t('viewAll')} ({countStories})
+                        {i18n.t('viewAll')} ({storiesCommunity?.length})
```

Comment:
Is there any chance that this shows "undefined" to the user if the internet is too slow?

Reply:
If the internet is off right after the user log in (highly unprovable, but can happen), yes.

Reply:
So, we rather show "0" than "undefined"